sympy/series/tests/test_limits.py

```diff
@@ -156,7 +156,7 @@ def test_floor():
 
     # https://github.com/sympy/sympy/issues/14478
     assert limit(x*floor(3/x)/2, x, 0, '+') == Rational(3, 2)
-    assert limit(floor(x + 1/2) - floor(x), x, oo) == AccumBounds(-0.5, 1.5)
+    assert limit(floor(x + 1/2) - floor(x), x, oo) == AccumBounds(-S.Half, S(3)/2)
```

Comment:
I think this should really have been `S(1)/2` or `S.Half` to start with. Nothing in the issue seems to indicate that this was an issue.

Reply:
I know it is not related to the thing you changed though, but still...

Reply:
The limit currently matches with [wolfram alpha](https://www.wolframalpha.com/input/?i=limit%28floor%28x+%2B+1%2F2%29+-+floor%28x%29%2C+x%2C+oo%29), though I am not sure why this result is so... I would have thought it should be `AccumBounds(0, 1)`.

Reply:
After going through the comments on the PR, this limit seems to be the cause of the other linked issue (#14313) (Ref this [comment](https://github.com/sympy/sympy/pull/20803#discussion_r560066567))